asset/css/variables.less

```diff
@@ -100,6 +100,13 @@
 
 @card-border-color: @base-gray-light;
 
+@crontab-disabled-text-color: @base-disabled;
+@crontab-selectable-list-bg: #404d72;
```

Comment:
`@searchbar-bg` uses the same color code. I think it's time now for a `@default-input-bg`.